src/_utils/interfaces.ts

```diff
@@ -0,0 +1,10 @@
+export interface CommonFieldsProps {
```

Comment:
You didn't remove it from TextField?

Reply:
I especially focus on this new component but yes it's not a big deal to use this utils with the TextField component.